platformio.ini

```diff
@@ -33,7 +33,7 @@ lib_deps =
   WebServer
   https://github.com/tzapu/wifiManager
   https://github.com/dojyorin/arduino_base64
-  https://github.com/OXRS-IO/Ethernet
+  https://github.com/austinscreations/wt32-sc01-ethernet
```

Comment:
I would prefer to merge your changes back into the OXRS ethernet fork - this will become difficult to maintain having a separate version of the lib for the touch panel f/w compared to everything else.

But it needs proper testing on the other firmware before we can do that safely.